templates/docs/examples/patterns/pagination/pagination-verbose.html

```diff
@@ -0,0 +1,10 @@
+{% extends "_layouts/examples.html" %} {% block title %}Pagination / Previous and next buttons only {% endblock %} {% block standalone_css %}patterns_pagination{% endblock %} {% block content %}
```

Comment:
It looks like you've run prettier on this.
Could you format it as in other examples (so the tags are in separate lines)?